app/uk/gov/hmrc/serviceconfigs/model/SlugInfo.scala

```diff
@@ -112,12 +117,14 @@ trait ApiSlugInfoFormats {
     )(SlugInfo.apply, unlift(SlugInfo.unapply))
   }
 
-  val dcFormat: OFormat[DependencyConfig] =
+  implicit val dcFormat: OFormat[DependencyConfig] =
```

Comment:
The reason this wasn't implicit is that it is implicit for all clients where DependencyConfig is in scope. I prefer clients to explicitly put implicits in scope as required (or create an Implicits subtrait which they explicitly import)
Here, `val slugFormat: OFormat[SlugMessage] = {
  implicit val dcf = dcFormat
  Json.format[SlugMessage]
}
`